hack/monitor_prow.py

```diff
@@ -75,7 +76,10 @@ def highlight(log_dir, dc):
             debug("deployment/{}: pod/{}: getting logs".format(dc, pod))
             try:
                 for l in run_oc(cmd).splitlines():
-                    if warn in l:
+                    new_line, replaced = replace(l)
+                    if replaced:
+                        log_lines.append(GRAY + new_line + RESET)
```

Comment:
I don't think we want to print anything at all here, or it's just more spam

Reply:
Done.